blueprints/third-party-solutions/f5-bigip/f5-bigip-ha-active-deployment/main.tf

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id = (
+    var.project_create
+    ? module.project.project_id
+    : var.project_id
+  )
+}
+
+data "google_client_openid_userinfo" "provider_identity" {}
```

Comment:
Luca, can you make this optional so we can run tests (which will have to be added here)?

I also think that this is fine for testing, but not what you want if you deploy the blueprint for actual usage. In that case IAM is taken care of at the folder or project level, and would use groups anyway.